js/logo.js

```diff
@@ -12201,6 +12247,31 @@ function Logo () {
 
         return Number(a) * Number(b);
     };
+    /** 
+     * calculate euclidean distance between (cursor x, cursor y) and (mouse 'x' and mouse 'y')
+     * @privileged
+     * @param   a
+     * @param   b
+     * @param   c
+     * @param   d
+     * @returns {number}
+     */
+    this._docalculatedistance = function (a,b,c,d) {
+        if (typeof(a) === 'string' || typeof(b) === 'string' || typeof(c) === 'string' || typeof(d) === 'string') {
+            this.errorMsg(NANERRORMSG);
+            this.stopTurtle = true;
+            return 0;
+        }
+
+        if(a===b and c===d)
```

Comment:
and is Python. In JavaScript you need &&

Did you test this code?

Reply:
No, I have not tested this  code.

Reply:
Please do not submit PRs w/o first testing your code.

Reply:
I am sorry @walterbender .